docs/man/Dockerfile.5.md

```diff
@@ -179,6 +190,11 @@ or
  it as holding externally-mounted volumes from the native host or from other
  containers.
 
+**NOVOLUME**
+ --**NOVOLUME ["/data"]**
+ The **NOVOLUME** informs Docker that the image no longer want to
```

Comment:
The **NOVOLUME** instruction informs Docker that the images no longer wants to...